addons/website/tests/test_ui.py

```diff
@@ -192,3 +192,16 @@ def test_07_snippet_version(self):
 
     def test_08_website_style_custo(self):
         self.start_tour("/", "website_style_edition", login="admin")
+
+    def test_09_website_conditional_visibility(self):
+        self.start_tour('/', 'conditional_visibility', login='admin')
+        self.browser_js('/?utm_medium=Email', """
+                const el = document.getElementById('wrap').getElementsByClassName('s_text_image')[0]
+                const style = window.getComputedStyle(el)
+                if (style.display === 'none') {
+                    console.error('this element should be visible now')
+                } else {
+                    console.log('test successful')
+                }
```

Comment:
Even if it feels overkill, I would prefer no JS in a python file: create a tour with an unique step just for this check.